odoo/fields.py

```diff
@@ -272,6 +272,8 @@ def __init__(self, string=Default, **kwargs):
         args = {key: val for key, val in kwargs.items() if val is not Default}
         self.args = args or EMPTY_DICT
         self._setup_done = None
+        self.name = type(self).__name__
+        self.model_name = 'unknown_model'
```

Comment:
why not instead use `_unknown` as model_name?

See:

https://github.com/odoo/odoo/blob/7bd18ff93dc94b0170622660abbbaeb514b3be09/odoo/addons/base/models/ir_model.py#L86

https://github.com/odoo/odoo/blob/7bd18ff93dc94b0170622660abbbaeb514b3be09/odoo/fields.py#L2256